tests/data/ingress-mtls/policies/ingress-mtls-invalid.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: k8s.nginx.org/v1
+kind: Policy
+metadata:
+  name: ingress-mtls-policy
+spec:
+  ingredsssMTLS:
+    clientCertSecrettt: ingress-mtls-secret
+    verifyadClient: "on"
+    verifyDeaerpth: 1
```

Comment:
This is very minor, so feel free to ignore, but I think putting a small comment in the yaml here to specify the error would be a good idea.